### DISCUSSION TITLE
Colors?

### DISCUSSION THREAD

**Discussion by lbdyck** [charmbracelet/gum#478] [Q&A] [answered]
I admit I'm new to `gum` but where can I find out the color options?

Tyvm

**Comment by maaslalani** [comment]
Hey @lbdyck,

Most `gum` commands have styling options on them to set various colours, styles, and theming options. For example, you can customize the colours on `gum spin` like so:

```
gum spin --spinner.foreground 11 --title.foreground 12 -- sleep 5
```

The `--spinner.foreground` and `--title.foreground` flags both take colours, either in `1-255` colors or `#HEX` format (`#FFFFFF`). This is consistent with all other `gum` commands as well. You can find the flag names with `gum commmand --help`.

**Comment by lbdyck** [comment]
where can I find those color numbers/codes?

**Reply by lbdyck** [reply]
I found a few websites that have the color codes - thank you

**Reply by zen0bit** [reply, marked as answer]
Look for gcolor in my gumaker repo...

Prepared in gumaker, just not yet there
